acceptance/tests/agent/agent_can_reconnect_after_master_failure.rb

```diff
@@ -0,0 +1,57 @@
+require 'securerandom'
+test_name 'C97323: agents can reconnect after master crash/restart' do
+
+  username = 'er0ck'
+  teardown do
+    step 'stop puppet agent' do
+      on(agents, puppet_resource("service puppet ensure=stopped"))
+    end
+    step 'remove test user' do
+      on(agent, puppet_resource("user #{username} ensure=absent"))
```

Comment:
Probably want to allow exit 1 also, in case the user was not successfully created in the body of the test.